launch/tier4_perception_launch/launch/traffic_light_recognition/traffic_light.launch.xml

```diff
@@ -106,10 +108,12 @@
         <push-ros-namespace namespace="detection"/>
         <include file="$(find-pkg-share traffic_light_map_based_detector)/launch/traffic_light_map_based_detector.launch.xml">
           <arg name="input/camera_info" value="$(var input/camera_info)"/>
-          <arg name="min_timestamp_offset" value="-0.04"/>
           <arg name="expect/rois" value="expect/rois"/>
           <arg name="output/rois" value="$(var map_based_detector_output_topic)"/>
           <arg name="output/camera_info" value="camera_info"/>
+          <!-- This parameter should be configured differently for each camera considering their delay. -->
+          <arg name="min_timestamp_offset" value="-0.3"/>
```

Comment:
The value changed to `-0.3` here. Is this intentional?

Reply:
Sorry I didn't notice. Thank you for the review.

Reply:
Fixed: https://github.com/autowarefoundation/autoware.universe/pull/6200/commits/695af9aef66a63e01fc7230ec285fbc939fa5089